samples/sampler/stories/qa/DayPicker.js

```diff
@@ -48,3 +49,19 @@ ToTestGetSelectedDayString.parameters = {
 		hideNoControlsWarning: true
 	}
 };
+
+export const WithSelectedNumber = (args) => (
+	<Scroller>
+		<Heading size="small">
+			Change *selected* number.
+			<br />
+			Change locale to *es-ES* starting on Monday.
+		</Heading>
+		<Item label={getSelectedDayString(args['selected'])}>{'Selected Day'}</Item>
+		<DayPicker disabled selected={args['selected']} />
```

Comment:
Why is this component disabled?

Reply:
I think it won't change the value when `selected` prop is given. Please check.

Reply:
I will remove the entire code change of qa-sampler